src/staticanalysis/bot/static_analysis_bot/coverity/coverity.py

```diff
@@ -180,7 +180,19 @@ def return_issues(self, coverity_results_path, revision):
                 logger.info('Coverity did not find any issues')
                 return []
 
-            return [CoverityIssue(issue, revision) for issue in result['issues']]
+            return [CoverityIssue(issue, revision) for issue in result['issues'] if self._is_local_issue(issue)]
+
+    def _is_local_issue(self, issue):
```

Comment:
Great ! Now you can move `_is_local_issue` directly in the body of `return_issues`, this is the equivalent of a private method in C++, that's an [inner function](https://realpython.com/inner-functions-what-are-they-good-for/).

You should then remove `self` and just call `_is_local_issue(issue)`